app/assets/javascripts/manage/lib/setupDataTables.js

```diff
@@ -25,6 +25,11 @@ var setupDataTables = function() {
       { orderable: true, data: 'active' },
       { orderable: true, data: 'receive_weekly_report' },
       { orderable: true, data: 'created_at' },
+      { orderable: true, data: 'current_sign_in_at' },
+      { orderable: true, data: 'last_sign_in_at' },
+      { orderable: true, data: 'current_sign_in_ip' },
+      { orderable: true, data: 'last_sign_in_ip' },
+      { orderable: true, data: 'sign_in_count' },
```

Comment:
Could you add `visible: false` to each of these? That way they only show up if someone _really_ wants to see them